hex/helpers/lib/run.exs

```diff
@@ -5,13 +5,15 @@ defmodule DependencyHelper do
     |> run()
     |> case do
       {output, 0} ->
+        output = Base.decode64!(output)
```

Comment:
@TylerWitt Thanks for all your work on updating Elixir/OTP for dependabot :pray: 

Reporting back on an issue having after these changes...

![image](https://github.com/user-attachments/assets/2b07eb27-a50c-43cf-8676-56e24f3e2ccb)

Apparently getting `"\n"` out of the encoded parse_deps.exs and raising here when trying to decode back?

Reply:
See #11406--should fix the issue.